CHANGELOG.md

```diff
@@ -5,18 +5,24 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to the versioning scheme outlined in the [README.md](README.md).
 
-## Upcoming
+## [2.05.0.3.0]
 
 ### Added
+
 - Added prometheus output for "transactions in last block" (#3138).
 - Added envrionement variable STACKS_LOG_FORMAT_TIME to set the time format
-  stacks-node uses for logging.
+  stacks-node uses for logging. (#3219)
   Example: STACKS_LOG_FORMAT_TIME="%Y-%m-%d %H:%M:%S" cargo stacks-node
+- Added mock-miner sample config (#3225)
 
 ### Changed
+
 - Updates to the logging of transaction events (#3139).
+- Moved puppet-chain to `./contrib/tools` directory and disabled compiling by default (#3200)
+- Optimized mempool walk to speed up miner loop 6x (#3229)
```

Comment:
Should this be #3254 (the PR number) instead of the issue number?

Reply:
The perf improvement of PR https://github.com/stacks-network/stacks-blockchain/pull/3254 on mining latency is about 10%-20% on M1/SSD. I would describe the PR as directly improving the VM eval latency which indirectly improves mining latency.

Reply:
In hindsight, I don't know why i added this - there is no PR associated with that issue in this release. line will be removed.

Reply:
https://github.com/stacks-network/stacks-blockchain/commit/c835cf99b347c907319f6a99873ef43c4ceeeefe